tests/positive/files/link.go

```diff
@@ -122,6 +124,105 @@ func CreateSymlinkOnRoot() types.Test {
 	}
 }
 
+func WriteThroughRelativeSymlink() types.Test {
+	name := "Write Through Relative Symlink on the Root Filesystem"
+	in := types.GetBaseDisk()
+	out := types.GetBaseDisk()
+	// note this abuses the order in which ignition writes links and will break with 3.0.0
```

Comment:
Might be worthwhile to add a `TODO:` for easy grepping when branching for 3.0.0

Reply:
I could go either way. We'll need to audit the tests for 3.0.0 anyway and if it breaks we'll see this note.